app/main.py

```diff
@@ -1,4 +1,6 @@
 def copy_file(command: str):
+    if command[1] == command[2]:
```

Comment:
You've introduced it before you even split the command in parts. Move it down a little.